magefiles/magefile.go

```diff
@@ -713,3 +724,65 @@ func AppendFrameworkDescribeGoFile(specFile string) error {
 	return err
 
 }
+
+func newSprayProxy() *SprayProxy {
+	sprayProxyUrl := os.Getenv("QE_SPRAYPROXY_HOST")
+	sprayProxyToken := os.Getenv("QE_SPRAYPROXY_TOKEN")
```

Comment:
where to get the env values `QE_SPRAYPROXY_HOST` and `QE_SPRAYPROXY_TOKEN` from when running locally?

Reply:
when you run it locally, you doesn't need to set the both variables, because it doesn't need to register pac server to the global Sprayproxy server.